src/front/wgsl/number.rs

```diff
@@ -94,7 +91,7 @@ fn parse(input: &str) -> (Result<Number, NumberError>, &str) {
     /// returns `true` and consumes `X` bytes from the given byte buffer
     /// if the given `X` nr of patterns are found at the start of the buffer
     macro_rules! consume {
-        ($bytes:ident, $($($pattern:pat)|*),*) => {
+        ($bytes:ident, $($($pattern:pat_param)|*),*) => {
```

Comment:
I think what we actually want is to just replace the whole `$(pattern:pat_param)|*` with `$pattern:pat` (with corresponding changes in the replacement), since the effect the author was going for was actually an or-pattern.

Reply:
Done; added e0dcc726 to address this.